pages/booking/list.tsx

```diff
@@ -1,15 +1,24 @@
 import BookingCard from '../../components/booking/BookingCard'
 import BookingsFilter from '../../components/booking/BookingsFilter'
 import InfiniteScroll from 'react-infinite-scroll-component'
-import getBookingStatusTypes from '../../lib/get-booking-status-types'
 import getMorePageData from '../../lib/get-more-page-data'
+import prisma from '../../lib/prisma'
 
 import { useEffect, useState } from 'react'
 
-function BookingList() {
+export async function getServerSideProps() {
+  const statusTypes = await prisma.booking_status.findMany()
+  return { 
+    props: { 
```

Comment:
I don't think you need this field

Reply:
If I do "props:JSON.parse(JSON.stringify(statusTypes))" I get this error - SerializableError: Error serializing props returned from `getServerSideProps` in "/booking/list".
Reason: Props must be returned as a plain object from getServerSideProps: `{ props: { ... } }` (received: `[object Array]`).
    at Object.isSerializableProps (C:\Users\SAV\Desktop\SalesWell\SalesWell\node_modules\next\dist\lib\is-serializable-props.js:16:15)